content/intro-to-storybook/angular/en/deploy.md

```diff
@@ -72,13 +72,13 @@ Hooray! We published Storybook with one command, but manually running a command
 
 ## Continuous deployment with Chromatic
 
-Now that our project is hosted in a GitHub repository, we can use a continuous integration(CI) service to deploy our Storybook automatically. [GitHub Actions](https://github.com/features/actions) is a free CI service that's built into GitHub that makes automatic publishing easy.
+Now that we've hosted our project in a GitHub repository, we can use a continuous integration (CI) service to deploy our Storybook automatically. [GitHub Actions](https://github.com/features/actions) is a free CI service that's built into GitHub that makes automatic publishing easy.
 
 ### Add a GitHub Action to deploy Storybook
 
-In the root folder of our project, create a new directory called `.github` then create another `workflows` directory inside of it.
+In the root folder of our project, create a new directory called `.github`, then create another `workflows` directory inside of it.
 
-Create a new file called `chromatic.yml` like the one below. Replace `CHROMATIC_PROJECT_TOKEN` with your project token.
+Create a new file called `chromatic.yml` like the one below. Make sure to replace `CHROMATIC_PROJECT_TOKEN` with your project token.
```

Comment:
While the Chromatic token is _probably_ OK to commit publicly, it's much better practice to obscure via secrets. The code snippet already uses it: `${{ secrets.CHROMATIC_PROJECT_TOKEN }}` is a special syntax to pull the `CHROMATIC_PROJECT_TOKEN` from the repo's secrets. 

We should link to https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository with instructions to add the `CHROMATIC_PROJECT_TOKEN` secret.

Reply:
That was originally when that section was redone but cut down for brevity purposes. I can update the link and provide the one you've mentioned for clarity

Reply:
I didn't even see the aside, as it wasn't modified when I was reviewing originally. 🤦‍♂️ 

Still, I think it's important to remove the "replace with your project token" language, to avoid guiding people into accidentally committing a secret token.

Reply:
Let me take a look at it and push a commit.

Reply:
Updated with the latest commit.